### DISCUSSION TITLE
I took Monika out and now she won't load

### DISCUSSION THREAD

**Discussion by MakotoEi0626** [Monika-After-Story/MonikaModDev#10296] [QA - tech support + general]
I took Monika out, her file is in "characters", but when i open MAS she isn't there
![Screenshot (314)](https://github.com/Monika-After-Story/MonikaModDev/assets/172993345/588549c3-085d-42ae-975f-25bb1afccd4b)
![Screenshot (313)](https://github.com/Monika-After-Story/MonikaModDev/assets/172993345/3d7e6a30-5acb-42da-b520-27ccda9aca96)

**Comment by Booplicate** [comment]
[Restore a backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).
